ukis_pysat/utils.py

```diff
@@ -0,0 +1,30 @@
+import pystac
+from geojson import Feature
+from pystac import ProviderRole
+from pystac.extensions.sar import FrequencyBand, SarExtension, Polarization
+
+
+def fill_sar_ext(sar_ext: SarExtension, meta: Feature):
```

Comment:
do we really need/want this one? if we would stay consistent throughout the different platforms, we should also consider to use `eo:bands` for the multi-spectral platforms. also i am not so happy with the current placement of `producttype` inside the sar_ext. imu this should be a more general attribute and is not SAR specific. The discussion on this topic seems also not fully converged yet at specs level: https://github.com/radiantearth/stac-spec/issues/897